sourcegraph/sourcegraph
# bext: omit credentials in legacy extension bundles requests

Omits credentials when requesting legacy extension bundles from the legacy extensions bucket. 

Check out [this Slack thread](https://sourcegraph.slack.com/archives/C03TGHH1S3V/p1675855887893259) for more context.

https://user-images.githubusercontent.com/25318659/217836038-8158895d-77a0-4f6a-9387-b36f77237b56.mov

## Test plan
Tested manually (video attached):

- run pre-4.0 instance locally with `extensions.remoteRegistry` site config param enabled (`v3.43` on video)
- run browser extension locally with changes introduced by this PR
- set your local instance as Sourcegraph URL in the browser extension options window
- go to the code host and open file from synced with your instance repo 
- ensure code-intel popovers work
- ensure extensions bundles requested from the legacy extensions bundles are loaded successfully
 


## App preview:

- [Web](https://sg-web-taras-yemets-bext-omit.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
